### ISSUE TITLE
Save the date 🎂 - OpenFaaS 3rd birthday!

### ISSUE THREAD

**Issue by alexellis** [openfaas/faas#1405]
Join us for the OpenFaaS 3rd birthday with an online Community Meeting.

We'll have demos from #TeamServerless and an update on the year so far.

![QAfn79n- jpg-large](https://user-images.githubusercontent.com/6358735/71088225-de33c580-2195-11ea-802d-0b1268bca410.jpeg)

## Date: 18th Dec 2019 @ 1700 GMT+0

Join the event via Google Calendar

https://calendar.google.com/event?action=TEMPLATE&tmeid=cjBtOHZicmpuaG9kczZ0cXRsOWEwNWI4cHMgb3BlbmZhYXMuY29tX2xvb2g2cW1kNTI1MXJyNGt0bjd0aTgwNHBrQGc&tmsrc=openfaas.com_looh6qmd5251rr4ktn7ti804pk%40group.calendar.google.com

Or on the day, via Zoom - 

https://zoom.us/j/727801872

Meeting ID: 727 801 872

Please +1 👍 if you're attending

And checkout [GitHub Sponsors for OpenFaaS to support Open Source in 2020.](https://insiders.openfaas.io/)

**Comment by kaihoffman** [comment]
I'm getting an error clicking on the calendar link saying "could not find event".

**Comment by alexellis** [comment]
Let me try again.

**Comment by alexellis** [comment]
Calendar is public as is the event.

**Comment by alexellis** [comment]
Catch up with the recording -> https://www.openfaas.com/blog/birthday-teamserverless/
